src/contracts/Airdrop.h

```diff
@@ -0,0 +1,226 @@
+using namespace QPI;
+
+#define AIRDROP_START_SUCCESS 0
+#define AIRDROP_INSUFFICIENT_FUND 1
+#define AIRDROP_STARTED 2
+#define AIRDROP_SEND_FUND 3
+#define AIRDROP_START_ERROR 4
+#define AIRDROP_INCORRECT_TIME 5
+#define AIRDROP_DISTRIBUTE_SUCCESS 6
+#define AIRDROP_DISTRIBUTE_FAILD 7
+#define AIRDROP_PERIOD_LIMITED 8
+#define AIRDROP_FUND_LIMITED 9
+#define AIRDROP_INSUFFICIENT_TOKEN 10
+
+struct AirdropLogger
+{
+    id src;
+    id dst;
+    sint64 amt;
+    uint32 logtype;
+    char _terminator;
+};
+
+struct AIRDROP2
+{
+};
+
+struct AIRDROP
+{
+private:
+    struct _AssetOrder
+	{
+		id entity;
+		sint64 numberOfShares;
+	};
+	collection<_AssetOrder, 2097152 * X_MULTIPLIER> _assetOrders;
```

Comment:
I know, VS does not understand our contract system and sometimes destroys code formatting, but please try to indent the code as it is common practice for C++ to improve readability.